scripts/asm-installer/cloudbuild.yaml

```diff
@@ -304,6 +310,7 @@ substitutions:
   _CLUSTER_LOCATION: us-central1-c
   _IMAGE_NAME: gcloud-with-kpt
   _SCRIPT_NAME: install_asm
+  _KPT_BRANCH: ""
```

Comment:
_ASM_PKG_BRANCH? _KPT_BRANCH is vague for others to understand.

Reply:
Good idea, done.